samples/kaspresso-sample/src/androidTest/kotlin/com/kaspersky/kaspressample/device_tests/DeviceLogcatSampleTest.kt

```diff
@@ -16,6 +18,8 @@ class DeviceLogcatSampleTest : TestCase() {
 
     @Test
     fun logcatTest() {
+        Assume.assumeTrue(Build.VERSION.SDK_INT != Build.VERSION_CODES.O) // "logcat -c" fails on android 8 for some reason
```

Comment:
We should fix it or describe this limitation in the docs (and throw an exception)